packages/components/menu/src/menu-item.ts

```diff
@@ -136,8 +151,13 @@ export class MenuItem extends ScopedElementsMixin(LitElement) {
        */
       setTimeout(() => this.#showSubMenu(), 100);
     } else if (this.selectable) {
-      this.selected = !this.selected;
-      this.selectEvent.emit(this.selected);
+      if (
+        !this.parentElement?.matches('[selects="single"]') ||
```

Comment:
Move `this.parentElement?.matches('[selects="single"]')` into a variable so you don't have to repeat yourself?